modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+* *Potentially easier collaboration in community and between content repositories.* For repositories that are single-sourced with community projects or that need to reuse content between product documentation repositories, this structure enables you to isolate the enterprise title compilation requirements so that the standalone components can be shared more readily. However, single-sourcing between community and enterprise content repositories poses other challenges beyond the scope of this document. (See the disadvantages and community considerations sections that follow.)
+* *Consistency with Pantheon 2 YAML configurations.* In this structure, the division of modules, assemblies, and other resources is consistent with the division of components in the `pantheon2.yml` file that is required for documents published on Pantheon 2.
+
+This repository structure poses the following disadvantages:
+
+* *Extensive use of symlinks.* In order to enable titles to be built with distinct assemblies, modules, and images, this repository structure uses many symlinks (symbolic links) for access across directories. You can extract these symlinks with a build script or additional variable configurations (such as `imagesdir`), but because this example repository is intended for a general CCS audience, it maintains explicit symlinks so that a user can build the repository titles with standard CCS and Asciidoctor tooling. Alternative repository structures that combine all relevant images, modules, and assemblies together in title folders can reduce the number of symlinks, but these alternative structures then forego many of the advantages listed previously.
+* *Potential conflicts with community content strategy.* Although this documentation repository structure is typically more collaboration-friendly than other FCC repository alternatives (as described in the previous advantages), the FCC strategy as a whole still might not be well received among all community projects. For example, many community projects prefer Mavenized repositories divided into chapter files instead of module files, all categorized in folders by guides or pages (titles) to be built. A community aversion to modularization in general can require writers to modify this example repository design, to pursue tooling alternatives to accommodate all content needs, or in some cases to abandon single-sourcing with community.
```

Comment:
Redundant references to (internal) FCC.

The whole paragraph is a speculation. I don't think it needs to be here.

Reply:
Right, understood. I explained to the group in the call about this that a recurring complaint I keep receiving in my courses and work with other teams is that the mod doc ref guide is great at explaining what to do, but not really why. Teams with differing strategies and opinions are eager for the rationale in some things, particularly with repo design. So in my initial research and planning for this repo example and documentation, one big request was to please explain the advantages and disadvantages of the general strategy here, since it's up to individual teams to assess and decided, based on their many factors. So some of this is indeed speculation and potential, but that's what I've been asked by many to provide. So I'd prefer to keep it as an answer to some of the potential why's.